modules/op-nonroot-buildah-user-namespaces.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+// * secure/unprivileged-building-of-container-images-using-buildah.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nonroot-buildah-user-namespaces_{context}"]
+= Running Buildah as a non-root user by configuring user namespaces
+
+Configuring user namespaces is the simplest way to run Buildah in a task as a non-root user. However, some images might not build using this option.
+
+.Prerequisites
+
+* You have installed the `oc` command-line utility.
+
+.Procedure
+
+. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace; change the name of the copy to `buildah-as-user`.
++
+[source,terminal]
+----
+$ oc get task buildah -n openshift-pipelines -o yaml | yq '. |= (del .metadata |= with_entries(select(.key == "name" )))' | yq '.kind="Task"' | yq '.metadata.name="buildah-as-user"' | oc create -f -
+----
+
+. Edit the copied `buildah` task by running the following command:
++
+[source,terminal]
+----
+$ oc edit task buildah-as-user
+----
++
+In the new task, create `annotations:` and `stepTemplate` sections, as shown in the following example:
++
+[source,yaml]
```

Comment:
just to be a little more explicit maybe something like:
```suggestion
.Example `buildah-as-user` task
[source,yaml]
```

Reply:
also i see the inline comment in this example file, and it could benefit from becoming an annotation instead